osscluster.go

```diff
@@ -1572,6 +1600,18 @@ func (c *ClusterClient) mapCmdsBySlot(cmds []Cmder) map[int][]Cmder {
 	}
 	return cmdsMap
 }
+func (c *ClusterClient) keyedAndKeyessCmds(cmds []Cmder) ([]Cmder, []Cmder) {
+	keyedCmds := make([]Cmder, 0, len(cmds))
+	keylessCmds := make([]Cmder, 0, len(cmds))
+	for _, cmd := range cmds {
+		if cmdFirstKeyPos(cmd) == 0 {
+			keylessCmds = append(keylessCmds, cmd)
+		} else {
+			keyedCmds = append(keyedCmds, cmd)
+		}
+	}
+	return keyedCmds, keylessCmds
+}
```

Comment:
This can be optimized to just extract the `keyedCmds` since it doesn't look at the keyless cmds at the moment. Will rewrite if the CI passes.